### PR TITLE
Update README.md to include adding a model to Model Registry

### PR DESCRIPTION
**Pull request by aguschin**
part of https://github.com/iterative/example-repos-dev/issues/203

### CODE REVIEW

**Review by daavoo — COMMENTED** [review]
Should we do this manually for the example repos?

**Comment by daavoo** [comment]
> howing Studio in the PR [iterative/example-get-started-experiments#126](https://github.com/iterative/example-get-started-experiments/pull/126), which is important enough IMO. Once we implement Studio API for adding models, we can make that part of the generation script.

So the idea is that in the example Repo the P.R. would be left open?

File: example-get-started-experiments/code/README.md [hunk 1]
```diff
@@ -108,6 +108,17 @@ This tag also contains a GitHub Actions workflow that reruns the pipeline if any
 [CML](https://cml.dev/) is used in this workflow to provision a cloud-based GPU 
 machine as well as report model performance results in Pull Requests.
 
+## Deploying the model
+
+Check out the [PR](https://github.com/iterative/example-get-started-experiments/pulls)
+that adds this model to 
+[Iterative Studio Model Registry](https://dvc.org/doc/studio/user-guide/model-registry/what-is-a-model-registry).
+You can [trigger CI/CD](https://dvc.org/doc/studio/user-guide/model-registry/use-models)
```
Comment (shcheklein) [inline example-get-started-experiments/code/README.md:116]: Can we put a link to the GH action file here?

**Review by shcheklein — APPROVED** [review]
just one minor suggestion, LGTM otherwise. Thanks, Alex.

File: example-get-started-experiments/code/README.md (hunk 1 above)
Comment (aguschin) [reply example-get-started-experiments/code/README.md:116]:
```suggestion
You can [trigger CI/CD](https://dvc.org/doc/studio/user-guide/model-registry/use-models#deploying-and-publishing-models-in-cicd)
```

Comment (aguschin) [reply example-get-started-experiments/code/README.md:116]: Putting a link to the section that explains CI/CD (forgot to add reference to exact part of the page there).

**Pull request merged**